docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
+
+For more information, including installation, testing and usage instructions, visit the `Pure Storage Flocker Driver on GitHub <https://github.com/sdodsley/purestorage-flocker-driver>`_
```

Comment:
We tested with 1.12.0, and with master of the Flocker tree as of a few days ago. I think the 0.3.2 was a remnant of some old doc links we had.